CHANGES.md

```diff
@@ -34,6 +34,7 @@ and `technomancy` for the above explanation.
 - Bump [pomegranate](https://github.com/cemerick/pomegranate) and [dynapath](https://github.com/tobias/dynapath) to `1.0.0`. [#612][612]
 - Digest `java.io.File` instead `String` path of jar at sift-action `:add-jar` method [#678][678]
 - Add warning about improper use of repl :eval option [#666][666]
+- Parse complex optargs with sequential types [682][682]
```

Comment:
Maybe add an example here what was previously possible and whats possible now.

Reply:
Ok I will add one